resource/environment/integration.js

```diff
@@ -159,20 +155,15 @@ const upgradeStack = Promise.coroutine(function* ( slack, envelope ) {
 	}
 
 	//get the template
-	const template = yield getTemplate( githubToken, githubOwner, githubRepo, branch, catalogNum);
+	const template = yield getTemplate( githubToken, githubOwner, githubRepo, branch, catalogNum).catch( () => undefined );
 	if ( template === undefined ) { 
-		return {
-			status: 404,
-			data: {
-				message: "Unable to get information from github"
-			}
-		};
+		return {status: 404, data: {message: "Unable to get information from github"}};
 	}
 	
 	//get the environments
 	const storedEnvironments = yield environment.getAll().catch( () => undefined );
 	if ( storedEnvironments === undefined ) { 
-		return {status: 404, data: {message: "Unable to get information from the database"}}; 
+		return {status: 500, data: {message: "Unable to get information from the database"}}; 
```

Comment:
Error messaging needs to be more helpful for the user. What happened? Is there something the user needs to do?, etc